packages/react-components/react-shared-contexts/src/BackgroundAppearanceContext/BackgroundAppearanceContext.ts

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+/**
+ * @internal
+ */
+export type BackgroundAppearanceContextValue = 'inverted' | undefined;
```

Comment:
I suppose it would require you to memoize the context value object so it is stable between renders, which adds complexity. Maybe not worth it in this case...

Reply:
I don't think it's worth it in this case, contexts are cheap, Our API patterns always use `appearance` as a string prop which is consistent.